tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/VarianceScaling.java

```diff
@@ -188,14 +188,14 @@ public VarianceScaling(Ops tf, double scale, Mode mode, Distribution distributio
   }
 
   /** The mode to use for calculating the fan values. */
-  public static enum Mode {
+  public enum Mode {
     FAN_IN,
     FAN_OUT,
     FAN_AVG
   }
 
   /** The random distribution to use when initializing the values. */
-  public static enum Distribution {
+  public enum Distribution {
     TRUNCATED_NORMAL,
     UNTRUNCATED_NORMAL,
```

Comment:
I'd missed this last time, could this just be `NORMAL` not `UNTRUNCATED_NORMAL`?

Reply:
OK